docs/apache-airflow-providers-tabular/index.rst

```diff
@@ -58,36 +58,63 @@
     Detailed list of commits <commits>
 
 
-Package apache-airflow-providers-tabular
+apache-airflow-providers-tabular package
 ------------------------------------------------------
 
 `Tabular <https://tabular.io/>`__
 
 
-Release: 1.5.0
+Release: 1.4.1
```

Comment:
1.5.0 is the latest. Probably worth regenerating after we've reverted the docs changes from the iceberg commit.

Reply:
Yeah. I think the `__init__.py` removal from tabulrar has hidden some other problems :).